docs/wfc3tools/pstat.rst

```diff
@@ -4,25 +4,87 @@
 pstat
 *****
 
-Plot statistics for a specified image section  up  the  stack  of  a
+Plot statistics for a specified image section  up  the  stack  of
 an IR MultiAccum image.  Sections from any of the SCI,
 ERR, DQ,  image extensions can be  plotted.   A  choice
-of  mean,  median,  mode,  standard  deviation,  minimum and maximum
+of  mean,  median,  mode,  standard  deviation,  minimum, and maximum
```

Comment:
function docstring implies that available stat is `midpt` rather than `median`.  Should check code to determine which is correct.